templates/publisher/listing.html

```diff
@@ -543,6 +561,9 @@
         {% for screenshot_url in screenshot_urls %}
           { url: {{ screenshot_url|tojson }}, type: "screenshot", status: "uploaded" },
         {% endfor %}
+        {% for banner_url in banner_urls %}
+          { url: {{ banner_url|tojson }}, type: "screenshot", status: "uploaded", is_banner: true },
```

Comment:
Nitpicking a bit, but as `is_banner` is a JS property maybe making it camelCase `isBanner` would make sense?

Reply:
:+1: I guess because I was doing it in the template I automatically went into pymode